nvidia_rag_mcp/mcp_client.py

```
@@ -105,6 +115,56 @@ async def _open_connection(ns: argparse.Namespace):
     For SSE: Connects to remote/local HTTP server with automatic endpoint probing.
     For streamable_http: Connects to FastMCP streamable-http endpoint.
     """
+
+    if ns.transport == "stdio":
+        try:
+            from mcp.client.stdio import stdio_client
+            try:
+                from mcp.client.stdio import StdioServerParameters  # type: ignore
+            except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)